plug.vim

```diff
@@ -110,18 +116,50 @@ let s:TYPE = {
 let s:loaded = get(s:, 'loaded', {})
 let s:triggers = get(s:, 'triggers', {})
 
+if s:is_win
+  function! s:plug_call(fn, ...)
+    let shellslash = &shellslash
+    try
+      set noshellslash
```

Comment:
I wondier if this is enough because https://github.com/vim/vim/commit/50f91d22bd81819d5d946f6c38252c7922120ec6 mentions that `completeslash` affects `globpath`. Could it be an issue for fzf.